content/en/docs/Examples/services.md

```diff
@@ -0,0 +1,409 @@
+    web_url: https://github.com/mailcow/mailcow-dockerized/releases/tag/{{ version }}
+    semantic_versioning: false
+    regex_version: ^[0-9-]+$
+    icon: https://raw.githubusercontent.com/mailcow/mailcow-dockerized/master/data/web/img/cow_mailcow.svg
+```
+
+## matomo-org/matomo
+Source: https://github.com/matomo-org/matomo
+```yaml
+service:
+  matomo-org/matomo:
+    type: github
+    url: matomo-org/matomo
+    regex_version: ^[0-9.]+$
+    web_url: https://github.com/matomo-org/matomo/releases/tag/{{ version }}
+    icon: https://raw.githubusercontent.com/matomo-org/matomo/4.x-dev/plugins/CoreHome/images/applogo_732.png
+```
+
+## mattermost/mattermost-server
+Source: https://github.com/mattermost/mattermost-server
+```yaml
+services:
+  mattermost/mattermost-server:
+    type: url
+    url: https://mattermost.com/deploy/
+    url_commands:
+      - type: regex_submatch
+        regex: releases\.mattermost\.com\/([^\/]+)\/mattermost-[0-9.]+-linux\-amd64\.tar\.gz
+    web_url: https://docs.mattermost.com/install/self-managed-changelog.html
+```
+
+## netbox-community/netbox
+Source: https://github.com/netbox-community/netbox
+```yaml
+service:
+  netbox-community/netbox:
+    type: github
+    url: netbox-community/netbox
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    regex_version: ^[0-9.]+$
+    web_url: https://github.com/netbox-community/netbox/releases/tag/v{{ version }}
+    icon: https://github.com/netbox-community/netbox/raw/develop/netbox/project-static/img/netbox_logo.svg
+    deployed_version:
+      url: https://netbox.lars-lehmann.net/
+      regex: '[0-9a-z]+ \(v([0-9.]+)\)'
+```
+
+## nextcloud/server
+Source: https://github.com/nextcloud/server
+```yaml
+services:
+  nextcloud/server:
+    type: github
+    url: nextcloud/server
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://nextcloud.com/changelog/#latest{{ version | split:"." | slice:":1" | join:""  }}
+    icon: https://github.com/nextcloud/server/raw/master/core/img/favicon.png
+    deployed_version:
+      url: https://nextcloud.example.io/status.php
+      json: versionstring
+```
+
+## prometheus/alertmanager
+Source: https://github.com/prometheus/alertmanager
+```yaml
+service:
+  prometheus/alertmanager:
+    type: github
+    url: prometheus/alertmanager
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://github.com/prometheus/alertmanager/blob/main/CHANGELOG.md
+    regex_content: alertmanager-{{ version }}\.linux-amd64.tar.gz
+```
+
+## prometheus/prometheus
+Source: https://github.com/prometheus/prometheus
+```yaml
+services:
+  prometheus/prometheus:
+    type: github
+    url: prometheus/prometheus
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://github.com/prometheus/prometheus/blob/main/CHANGELOG.md
+    regex_content: prometheus-{{ version }}\.linux-amd64
```

Comment:
Not sure whether you want `deployed_version` everywhere we know a url, but I think we should? Here's the Prometheus one
```
    deployed_version:
      url: https://prometheus.example.io/api/v1/status/buildinfo
      json: version
```
(same for Thanos, just thanos in the url)

Reply:
Yes would enter the `deployed_version` everywhere we have a URL for it.